libs/ui-lib/lib/ocm/components/clusterConfiguration/platformIntegration/PlatformIntegrationNote.tsx

```diff
@@ -10,14 +10,16 @@ const PlatformIntegrationNote = ({ platformType }: { platformType: SupportedPlat
     <p>
       <ExclamationTriangleIcon color={warningColor.value} size="sm" /> You will need to modify your
       platform configuration after cluster installation is completed.{' '}
-      <a
-        href={integrationPlatformLink || ''}
-        target="_blank"
-        rel="noopener noreferrer"
-        data-ouia-component-id="vm-integration-kb-page"
-      >
-        Learn more <i className="fas fa-external-link-alt" />
-      </a>
+      {!!integrationPlatformLink && (
+        <a
+          href={integrationPlatformLink || ''}
+          target="_blank"
+          rel="noopener noreferrer"
+          data-ouia-component-id="vm-integration-kb-page"
+        >
+          Learn more <i className="fas fa-external-link-alt" />
+        </a>
```

Comment:
Can we perhaps replace the `a` element with our `ExternalLink` component here? :slightly_smiling_face:

Reply:
Done!

Reply:
But please, keep the `data-ouia-component-id` - it's used for tracking in Pendo.

Reply:
This property doesn't exist in ExternalLink component